sources/packages/backend/apps/api/src/utilities/upload-utils.ts

```diff
@@ -3,9 +3,9 @@ import * as path from "path";
 
 const MAX_FILE_SIZE = +process.env.FILE_UPLOAD_MAX_FILE_SIZE;
 const ALLOWED_FILE_EXTENSIONS =
-  process.env.FILE_UPLOAD_ALLOWED_EXTENSIONS.split(",").map((value) =>
+  process.env.FILE_UPLOAD_ALLOWED_EXTENSIONS?.split(",").map((value) =>
```

Comment:
I believe the null check not being in place is failing the local test cases.

Reply:
It was in the past. It was fixed when monorepo changes were done. All tests/e2e tests must work locally.
This causes an issue while running the POD when the `upload-utils` is not really needed but the missing env variable makes the code crash and causes a looping in the POD startup.